analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -261,6 +282,39 @@ private static void VerifyBuildErrors(ImmutableArray<Diagnostic> diagnostics, Co
 
         private static string IssueType(bool isPrimary) => isPrimary ? "primary" : "secondary";
 
+        internal class File
+        {
+            private string FileName { get; }
+            private SourceText SourceText { get; }
```

Comment:
This doesn't have to be field now, as it's hidden. I've also suggested to rename the second one.
```suggestion
            private readonly string fileName;
            private readonly SourceText content;
```

Reply:
> I've also suggested to rename the second one.

😃

Reply:
Done :)